recipe/build.sh

```diff
@@ -1,18 +1,56 @@
-#!/bin/bash
-# Get an updated config.sub and config.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.* .
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
 
 export XERCESCROOT=${PREFIX}
 export XALANCROOT=${SRC_DIR}
 
-if [[ ${target_platform} == osx-64 ]]; then
-    platform=macosx
-else
-    platform=linux
-    export CXXCPP=${CPP}
+sed -i 's/CMAKE_CXX_STANDARD 14/CMAKE_CXX_STANDARD 17/' CMakeLists.txt
+
+if [[ ${CONDA_BUILD_CROSS_COMPILATION:-0} == 1 ]]; then
+    BOOTSTRAP_CMAKE_ARGS=${CMAKE_ARGS//${PREFIX}/${BUILD_PREFIX}}
+    BOOTSTRAP_CMAKE_ARGS=${BOOTSTRAP_CMAKE_ARGS//${CONDA_TOOLCHAIN_HOST}/${CONDA_TOOLCHAIN_BUILD}}
+
+    CROSS_LDFLAGS=${LDFLAGS}
+    CROSS_CC="${CC}"
+    CROSS_CXX="${CXX}"
+    CROSS_LD="${LD}"
+    CROSS_PKG_CONFIG_PATH=${PKG_CONFIG_PATH}
+
+    LDFLAGS=${LDFLAGS//${PREFIX}/${BUILD_PREFIX}}
+    CC=${CC//${CONDA_TOOLCHAIN_HOST}/${CONDA_TOOLCHAIN_BUILD}}
+    CXX=${CXX//${CONDA_TOOLCHAIN_HOST}/${CONDA_TOOLCHAIN_BUILD}}
+    LD="${LD//${CONDA_TOOLCHAIN_HOST}/${CONDA_TOOLCHAIN_BUILD}}"
```

Comment:
```suggestion
    CC=${CC_FOR_BUILD}
    CXX=${CXX_FOR_BUILD}
    unset LD
```

Reply:
This helped to clean up the recipe but unfortunately we're still stuck with incorrect linkage for the bootstrap.  I've posted the errors in a separate comment.